./run_kubernetes.sh
Error from server (AlreadyExists): deployments.apps "ml-api" already exists
NAME                      READY   STATUS    RESTARTS   AGE
ml-api-6b57555f8b-f7lkv   1/1     Running   0          3m17s
Forwarding from 127.0.0.1:8000 -> 80
Forwarding from [::1]:8000 -> 80
Handling connection for 8000

kubectl logs deployment/ml-api
 * Serving Flask app "app" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on http://0.0.0.0:80/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 231-779-307
[2020-04-06 06:38:54,245] INFO in app: JSON payload:
{'CHAS': {'0': 0}, 'RM': {'0': 6.575}, 'TAX': {'0': 296.0}, 'PTRATIO': {'0': 15.3}, 'B': {'0': 396.9}, 'LSTAT': {'0': 4.98}}
[2020-04-06 06:38:54,262] INFO in app: Inference payload DataFrame:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-04-06 06:38:54,276] INFO in app: Scaling Payload:
   CHAS     RM    TAX  PTRATIO      B  LSTAT
0     0  6.575  296.0     15.3  396.9   4.98
[2020-04-06 06:38:54,280] INFO in app: output prediction: [20.35373177134412]
127.0.0.1 - - [06/Apr/2020 06:38:54] "POST /predict HTTP/1.1" 200 -